pyqtgraph/graphicsItems/ImageItem.py

```diff
@@ -323,6 +324,24 @@ def setAutoDownsample(self, active: bool=True):
         self._renderRequired = True
         self.update()
 
+    def setNanPolicy(self, nanPolicy: str):
+        """
+        Control how NaN values are handled during downsampling for this ImageItem.
+
+        Parameters
+        ----------
+        nanPolicy : str
+            Must be one of ['ignore`, 'propagate']. If 'nanPolicy' is 'ignore', NaNs
+            are automatically ignored during downsampling, at the expense of
+            performance. If 'nanPolicy' is 'propagate', NaNs are kept during
+            downsampling. ImageItem is instantiated with nanPolicy set to 'ignore'.
```

Comment:
You've actually added a switch to make a new ImageItem with any nanPolicy. Maybe:
```python
Unless a different policy was specified, a new ImageItem is created with ``nanPolicy='propagate'``.
```